src/app.js

```diff
@@ -1,21 +1,34 @@
 'use strict';
+/* eslint-disable no-console */
 
-/**
- * Implement sum function:
- *
- * Function takes 2 numbers and returns their sum
- *
- * sum(1, 2) === 3
- * sum(1, 11) === 12
- *
- * @param {number} a
- * @param {number} b
- *
- * @return {number}
- */
-function sum(a, b) {
-  // write code here
-  return a + b;
-}
-
-module.exports = sum;
+const http = require('http');
+const fs = require('fs');
+
+const server = http.createServer((req, res) => {
+  const normalizedURL = new URL(req.url, `http://${req.headers.host}`);
+  let fileName = normalizedURL.pathname;
+
+  if (!fileName.startsWith('/file')) {
+    console.log('To load file write: "/file/-path-to-file-"');
+    res.end();
+  }
+
+  fileName = normalizedURL.pathname.replace(/\/file\/*/, '') || 'index.html';
```

Comment:
Add `^` token to the beginning of your regex, so it would match `/file/` only on the beginning of your string.
I know that it won't impact your functionality in this particular case, but better to have your regexes bomb-proof :bomb: :)
```suggestion
  fileName = normalizedURL.pathname.replace(/^\/file\/*/, '') || 'index.html';
```